diff --git a/dist/esm/src/client/appkit-base-client.js b/dist/esm/src/client/appkit-base-client.js
index f4079363dfd1f2f0fde59b2258e59b77be16fc5a..f77e32ab52b34bb1bac955a0368df1344f1ddae8 100644
--- a/dist/esm/src/client/appkit-base-client.js
+++ b/dist/esm/src/client/appkit-base-client.js
@@ -421,6 +421,7 @@ export class AppKitBaseClient {
         OptionsController.setEIP6963Enabled(options.enableEIP6963 !== false);
         OptionsController.setEnableInjected(options.enableInjected !== false);
         OptionsController.setEnableCoinbase(options.enableCoinbase !== false);
+        OptionsController.setEnableBaseAccount(options.enableBaseAccount !== false);
         OptionsController.setEnableNetworkSwitch(options.enableNetworkSwitch !== false);
         OptionsController.setEnableReconnect(options.enableReconnect !== false);
         OptionsController.setEnableMobileFullScreen(options.enableMobileFullScreen === true);
